WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -947,6 +948,12 @@ local function modify(parent, region, data)
     end
   end
 
+  if data.useAnchorPerUnit and data.anchorPerUnit == "UNITFRAME" then
+    LGF.RegisterCallback("WeakAuras", "GETFRAME_REFRESH", function()
+      region:PositionChildren()
+    end)
+  end
```

Comment:
You need to deregister in the else branch here.

Reply:
fixed

Reply:
I don't see a else branch.